src/main/java/gift/application/member/service/apicaller/KaKaoApiCaller.java

```diff
@@ -0,0 +1,92 @@
+package gift.application.member.service.apicaller;
+
+import static io.jsonwebtoken.Header.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_FORM_URLENCODED_VALUE;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import gift.global.validate.TimeOutException;
+import java.net.URI;
+import java.time.Duration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class KaKaoApiCaller {
+
+    private static final String KAKAO_USER_INFO_URL = "https://kapi.kakao.com/v2/user/me";
+    private static final String KAKAO_TOKEN_URL = "https://kauth.kakao.com/oauth/token";
+    private static final String GRANT_TYPE = "authorization_code";
+    private static final Duration TIMEOUT = Duration.ofSeconds(2);
+    @Value("${kakao.client_id}")
+    private String CLIENT_ID;
```

Comment:
해당 값은 상수가 아니라서요ㅎㅎ 다음과 같이 사용하면 될 것 같습니다!
SHIFT + F6으로 변경할 수 있습니다:)
```suggestion
    private String clientId;
```

Reply:
넵 수정하겠습니다!